docs/static/persistent-queues.asciidoc

```diff
@@ -42,7 +42,7 @@ TIP: If you want to define values for a specific pipeline, use <<multiple-pipeli
 
 `queue.type`:: Specify `persisted` to enable persistent queues. By default, persistent queues are disabled (default: `queue.type: memory`).
 `path.queue`:: The directory path where the data files will be stored. By default, the files are stored in `path.data/queue`. 
-`queue.page_capacity`:: The maximum size of a queue page in bytes. The queue data consists of append-only files called "pages". The default size is 64mb. Changing this value is unlikely to have performance benefits.
+`queue.page_capacity`:: The maximum size of a queue page in bytes. The queue data consists of append-only files called "pages". The default size is 64mb. Changing this value is unlikely to have performance benefits. You can change the page capacity of existing queue. The new size will apply to the new page.
```

Comment:
```suggestion
`queue.page_capacity`:: The queue data consists of append-only files called "pages." This value sets the maximum size of a queue page in bytes. 
The default size of 64mb is a good value for most users, and changing this value is unlikely to have performance benefits. 
If you change the page capacity of an existing queue, the new size applies only to the new page.
```

Reply:
@kaisecheng, I tried to smooth the flow of the wording a bit. Let me know what you think.